src/extension/src/scripts/run-tests.ts

```diff
@@ -0,0 +1,150 @@
+let child_process = require("child_process");
+const path = require("path");
+const fs = require("fs");
+
+const testFolder = path.join(__dirname, "..", "..", "src", "template_test");
+console.log(testFolder);
+let files: string[] = [];
+
+//Recursively delete all generated projects in template_test
+let deleteFolderRecursive = function(pathname: string) {
```

Comment:
I don't think this method is necessary. 

Our package.json has:
https://github.com/microsoft/WebTemplateStudio/blob/e540335646b1601e26efb120fd76a6c600cf9284/src/extension/package.json#L82

The same thing can be achieved by doing
```ts
const del = require("del");
del.sync(pathname);
```

Reply:
wow that actually cleans up my code so much

Reply:
Just added it, thanks for the suggestion!